tests/unittests/tests-nanocoap/tests-nanocoap.c

```diff
@@ -155,6 +155,10 @@ static void test_nanocoap__put_req(void)
     TEST_ASSERT_EQUAL_INT(fmt_opt_len, len);
     TEST_ASSERT_EQUAL_INT(COAP_FORMAT_TEXT, coap_get_content_type(&pkt));
 
+    len = coap_opt_add_uint(&pkt, COAP_OPT_ACCEPT, COAP_FORMAT_CBOR);
+    TEST_ASSERT_EQUAL_INT(fmt_opt_len, len);
```

Comment:
Seems, due to CBOR being chosen, the length of that option is longer now longer than 1.

```suggestion
    size_t accept_opt_len = 2;
    len = coap_opt_add_uint(&pkt, COAP_OPT_ACCEPT, COAP_FORMAT_CBOR);
    TEST_ASSERT_EQUAL_INT(accept_opt_len, len);
```

I would prefer the variable declaration to be moved up top, but the Github interface makes it a bit difficult to suggest this, so please adapt manually.